about.html

```diff
@@ -0,0 +1,166 @@
+            doloremque tempora voluptas maxime? Nulla ex, magni odit nihil
+            aspernatur itaque fugiat numquam odio veritatis placeat perferendis,
+            enim suscipit quas ipsa blanditiis! Et consequatur reiciendis
+            temporibus, animi earum ipsa id distinctio dolor tempora mollitia,
+            similique?
+          </p>
+          <small>
+            If you want a car, please contact using this email
+            www.hyundai.rw</small
+          >
+        </div>
+      </section>
+
+      <section id="courses" class="my-team">
+        <p>VISIT OUR BRAND ALL OVER THE WORLD</p>
+        <hr />
+        <div class="teamList">
+          <div class="listPara">
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod,
+              corrupti doloribus. Quam temporibus cum culpa fugiat. Quae aut
+              repudiandae magnam, ullam fuga sapiente, laboriosam expedita
+              voluptas natus explicabo reprehenderit esse. Quae, ipsum quaerat
+              modi facere harum nam recusandae voluptate nesciunt consectetur
+              amet enim corrupti delectus accusantium quasi dicta perferendis
+              veniam.repudiandae magnam, ullam fuga sapiente, laboriosam
+              expedita voluptas natus explicabo reprehenderit esse. Quae, ipsum
+              quaerat modi facere harum nam recusandae voluptate nesciunt
+              consectetur amet enim corrupti delectus accusantium quasi dicta
+              perferendis veniam.
+            </p>
+          </div>
+          <div class="brand">
+            <a
+              href="https://www.freepnglogos.com/pics/hyundai-logo"
+              title="Image from freepnglogos.com"
+              ><img
+                src="https://www.freepnglogos.com/uploads/hyundai-logo-0.png"
+                alt="hyundai logo emblem png"
+            /></a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <div class="marks">
+        <p>SEE OUR TEAM WORKING STRATEGY</p>
+        <hr />
+        <small> Lorem ipsum dolor sit amet consectetur adipisicing </small>
+        <small> Lorem ipsum dolor sit amet consectetur adipisicing </small>
+        <small> Lorem ipsum dolor sit amet consectetur adipisicing </small>
+
+        <ul>
+          <li>
+            <img src="images/toyota.svg" alt="hyundai car_logo" />
+          </li>
+          <li><img src="images/benz.png" alt="hyundai car_logo" /></li>
+        </ul>
+      </div>
+      <div>
+        <div class="myFooter">
+          <div class="brand">
+            <a
+              href="https://www.freepnglogos.com/pics/hyundai-logo"
+              title="Image from freepnglogos.com"
+              ><img
+                src="https://www.freepnglogos.com/uploads/hyundai-logo-0.png"
+                alt="hyundai logo emblem png"
+            /></a>
+          </div>
+          <p>
```

Comment:
- [ ] It would be nice to follow the design and use the `background-color: #272A31` specified for the footer. 

- [ ] For the mobile version, it would be nice to have the logo of the project included in the footer of this page.

### Your Design (Desktop Version)

![image](https://user-images.githubusercontent.com/77106962/153724532-c535dc22-3c54-4f64-912d-289603d08b26.png)

### Your Design (Mobile Version)

![image](https://user-images.githubusercontent.com/77106962/153724589-a751af80-91cc-499e-a7d0-7db8c8ba5fb3.png)

### Behance Design (Desktop Version)

![image](https://user-images.githubusercontent.com/77106962/153724624-dec23d24-a8b6-4842-a3db-18dcd45e33df.png)

### Behance Design (Mobile Version)

![image](https://user-images.githubusercontent.com/77106962/153724639-254fc20d-3ab5-449b-8941-9fe712a6bdd3.png)